src/pages/myGeotab/whatsNew.tsx

```diff
@@ -1,8 +1,3045 @@
+				<p>Nuget: Replace usage of Microsoft.Net.Http with System.Net.Http (WebRequest =&gt; HttpClient)</p>
+			</li>
+			<li>
+				<p>DiagnosticSearch: Added DiagnosticType property to search by the type of diagnostic. Ex, only GoFault diagnostics.</p>
+			</li>
+			<li>
+				<p>Added “ExpiredPasswordException” object. This exception can be thrown if a user makes a request while their ChangePassword flag is true. The user must change their password before they are able to successfully make further requests</p>
+			</li>
+			<li>
+				<p>SecurityRole: Added “ReportHosAvailability” <b><i>requires update of .net nuget package to ensure compatibility</i></b></p>
+			</li>
+			<li>
+				<p>If you are using the .net nuget package and plan to use the new “HOS Only” device plan you must update to nuget package version 5.7.1701 or greater to ensure compatibility.</p>
+			</li>
+		</ul>
+	</div>
+);
+
+const update1612: ReactNode = (
+	<div className="paragraph">
+		<ul>
+			<li>
+				<p>Id refactoring - The ID object has been refactored in the .NET SDK. See <a href="https://helpdesk.geotab.com/hc/en-us/community/posts/255601466--NET-SDK-ID-Object-Changes">this forum post</a> for details.</p>
+			</li>
+			<li>
+				<p>Updated description of GoCurveAuxiliary (GO4v3, GO6, GO7) properties: ImmobilizeUnit: With ImmobilizeUnit being true, it is used to define the delay before the driver identification reminder is sent out if the driver key has not been not swiped. The maximum value of this property is 255. When it is less or equal to 180, it indicates the number of seconds of the delay. When it is greater than 180, the delay increases 30 seconds for every increment of one of this property. For example, 180 indicates 180 seconds, 181 indicates 210 seconds, and 182 indicates 240 seconds. ImmobilizeArming: A value mainly used for enable or disable driver identification reminder. If it is used in conjunction with vehicle relay circuits, it can force the driver to swipe the driver key before starting the vehicle.</p>
+			</li>
+		</ul>
+	</div>
+);
+
+//ToDo: Update SDK Concepts link
+const update1611: ReactNode = (
+	<div className="paragraph">
+		<ul>
+			<li>
+				<p>Authentication rate limiting being phased in. See this <a href="https://www.geotab.com/blog/api-call-limits/">Blog Post</a> for more details. Added “Rate Limiting” section to <a href="/sdk/software/guides/concepts/">SDK Concepts</a>.</p>
+			</li>
+			<li>
+				<p>KnownId - Added: “DiagnosticDieselExhaustFluidId”, ”DiagnosticDieselParticulateFilterLampId”, “DiagnosticPowerTakeoffEngagedId”, “DiagnosticPowerTakeoffTotalFuelUsedId”</p>
+			</li>
+			<li>
+				<p>KnownId - Removed: “DiagnosticBluetoothBeaconOutOfRangeId”</p>
+			</li>
+			<li>
+				<p>Trailer - Added “Groups” property. Trailers can now be added to groups.</p>
+			</li>
+			<li>
+				<p>TrailerSearch - Added property groups. Search for Trailers that are members of these GroupSearch(s) one of it’s children or one of it’s parents.</p>
```

Comment:
The character U+2019 "’" could be confused with the ASCII character U+0060 "`", which is more common in source code.